demo/gdi/nuklear_gdi.h

```diff
@@ -218,27 +218,26 @@ nk_gdi_fill_rect(HDC dc, short x, short y, unsigned short w,
 static void
 nk_gdi_set_vertexColor(PTRIVERTEX tri, struct nk_color col)
 {
-    tri->Red   = col.r << 8;
-    tri->Green = col.g << 8;
-    tri->Blue  = col.b << 8;
-    tri->Alpha = 0xff << 8;
+    tri->Red   = col.r * (0xffff / 0xff);
+    tri->Green = col.g * (0xffff / 0xff);
+    tri->Blue  = col.b * (0xffff / 0xff);
+    tri->Alpha = col.a * (0xffff / 0xff);
 }
 
 static void
 nk_gdi_rect_multi_color(HDC dc, short x, short y, unsigned short w,
     unsigned short h, struct nk_color left, struct nk_color top,
     struct nk_color right, struct nk_color bottom)
 {
+    /* Note that GDI (without plus) does not appear to support alpha transparency properly in gradients. */
```

Comment:
Could you also add one or two words about how it manifests itself visually (I mean the "flickering")? Thanks!

Reply:
Sure, when hovering and moving the mouse over the color picker matrix, the displayed gradient flickers to black.
I've prepared a short video: https://www.youtube.com/watch?v=oaEzvqKD_fo